### PR TITLE
Pipeline GH Action for microbenchmarks

### PR DESCRIPTION
**Pull request by apanczyk**

### CODE REVIEW

**Comment by apanczyk** [comment]
Completed benchmark with SampleSpelBenchmark: https://github.com/apanczyk/nussknacker/actions/runs/1126883805
Failed benchmark scheduled at 00.00 UTC (already fixed): https://github.com/apanczyk/nussknacker/actions/runs/1118308151

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Benchmarks
+on:
+  schedule:
+#   Cron uses UTC - now this time is 00:00, during winter it won't be
```
Comment (mproch) [inline .github/workflows/benchmark.yml:4]: let's remove this comment, we don't really care about exact hour...

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "openjdk@1.11"
+      - name: Run benchmarks
+        run: sbt "benchmarks/jmh:run -i 8 -wi 3 -f1 -t2 .SampleSpelBenchmark -rf csv"
```
Comment (mproch) [inline .github/workflows/benchmark.yml:19]: I wonder, maybe it's better to use just one thread? (-t1) 
And let's check how long will it take for all benchmarks to run?

Comment (apanczyk) [reply .github/workflows/benchmark.yml]: At first glance there are minor/no differences between -t1 or -t2. With same parameters full benchmarks should be completed about 40 minutes https://github.com/apanczyk/nussknacker/actions/runs/1127313049.

**Comment by dswiecki** [comment]
LGTM!

**Review by dswiecki — APPROVED** [review]

**Pull request merged**